### PR TITLE
SHS-5841: The collection title on a wider screen is misaligned

### PR DESCRIPTION
**Pull request by mariannuar**
# READY FOR REVIEW

## Summary
Fix the collection title alignment on a wider screen

## Need Review By (Date)
10/08

## Urgency
low

## Steps to Test
1. Add a Collection to a Flexible Page
2. Select the visible as a Heading 2 option under Title settings
3. Set Background Color (which is getting updated to Background Width) to Full Width
4. Verify the title is aligned to the rest of the content

Screenshot 2024-10-01 at 1 40 56 PM

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208463191591251

### CODE REVIEW

**Comment by ahughes3** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1640-ffprxqwde192ojtqv7cdprltesitnxzo.tugboatqa.com
- https://hs-colorful-ffprxqwde192ojtqv7cdprltesitnxzo.tugboatqa.com
- https://hs-traditional-ffprxqwde192ojtqv7cdprltesitnxzo.tugboatqa.com
- https://english-ffprxqwde192ojtqv7cdprltesitnxzo.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66fc5058365d2bbbe5b88109

Visual Diffs:
- https://dashboard.tugboatqa.com/66fc5058365d2bbbe5b88109/visualdiffs

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss [hunk 1]
```diff
@@ -54,7 +54,8 @@
     }
 
     .field-hs-row-components,
-    .field-hs-collection-items {
+    .field-hs-collection-items,
+    h2 {
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss]: Make sure not to affect other possible `<h2>` within the component:

```suggestion
    > h2 {
```

**Review by cienvaras — CHANGES_REQUESTED** [review]
@mariannuar Looks good! Just a minor change needed.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss (hunk 1 above)
Comment (mariannuar) [reply docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss]: Ahhh yess, nice catch!. Thanks!

**Comment by mariannuar** [comment]
@cienvaras Ready for review again!

**Review by cienvaras — APPROVED** [review]
@mariannuar Thanks! 🚀

**Comment by cienvaras** [comment]
@ahughes3 Ready for you to review.

**Review by ahughes3 — APPROVED** [review]

**Review by joegl — APPROVED** [review]

**Pull request merged**